data/data_loader.py

```diff
@@ -1,7 +1,44 @@
 import numpy as np
 import pandas as pd
-from typing import Optional
-from data.datasets import credit_card_default_loader
+from typing import Optional, Mapping, Tuple
+from data.datasets import credit_card_default_loader, base_loader
+import enum
+
+
+class DatasetName(enum.Enum):
```

Comment:
Here and in the other class and function docstrings, the ending three quotations are on the next line, e.g.

"""One line description.

...
<last line of comment text.>
"""

Reply:
Good catch! The next PR, PR 17, updates many of the docstrings. I'll correct the formatting in that PR to avoid conflicts.